src/workerd/api/tests/worker-test.js

```diff
@@ -0,0 +1,19 @@
+import { throws, rejects } from 'node:assert';
+
+// We allow eval and new Function() on bootstrap phase.
+eval('1+1'); // should work
```

Comment:
We should verify that the eval'd code actually ran. At present this test would pass if eval were implemented as a no-op.